packages/server/src/cli.ts

```diff
@@ -15,21 +15,22 @@ import { createApp } from "./server";
 import { forkModule, requireFork, requireModule } from "./vscode/bootstrapFork";
 import { SharedProcess, SharedProcessState } from "./vscode/sharedProcess";
 import opn = require("opn");
+import { args } from "@oclif/parser";
 
 export class Entry extends Command {
 	public static description = "Start your own self-hosted browser-accessible VS Code";
 	public static flags = {
-		cert: flags.string(),
-		"cert-key": flags.string(),
-		"data-dir": flags.string({ char: "d" }),
+		cert: flags.string({char: "c", description: "Point to an SSL certificate .crt file"}),
+		"cert-key": flags.string({ char: "k", description: "Point to an SSL certificate .key file"}),
```

Comment:
Should be TLS not SSL

TLS certificate key files don't have to end in .key

Maybe change to "Point to a TLS certificate private key file"